src/Libraries/PythonNodeModelsWpf/PythonNodeModelsWpf.csproj

```diff
@@ -97,6 +97,10 @@
       <Name>DynamoServices</Name>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\..\PythonMigrationViewExtension\PythonMigrationViewExtension.csproj">
+      <Project>{10AF430D-0D3A-49CE-A63D-848912959745}</Project>
+      <Name>PythonMigrationViewExtension</Name>
+    </ProjectReference>
```

Comment:
I agree this is not nice. We should be able to tunnel an event through the model that can reach the extension.

Reply:
yeah, I do not like this at all but failed to find a supporting workaround. Maybe event based would be better